test/sideloading_test.rb

```diff
@@ -143,4 +143,26 @@
       end
     end
   end
+
+  context 'support for include_[attrbute]' do
+    let(:relation)   { User.all }
+    let(:controller) { UserController.new }
+    let(:options)    { { each_serializer: UserSerializer } }
+
+    before do
+      user    = User.create name: 'John', mobile: "51111111"
+      address = Address.create district_name: "mumbai", user_id: user.id
+      @json_expected = "{\"users\":[{\"id\":#{user.id},\"name\":\"John\",\"mobile\":\"51111111\",\"offer_ids\":[],\"reviewed_offer_ids\":[]}],\"offers\":[],\"addresses\":[{\"id\":#{address.id},\"district_name\":\"mumbai\"}]}"
```

Comment:
You might as well just have these in the `it` blocks, as they are only used once each